cmd/internal/planetscale_edge_database.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/go-sql-driver/mysql"
```

Comment:
@lizztheblizz Looks like this was accidentally added?

Reply:
Huh. I don't remember touching that at all. Removing.